### ISSUE TITLE
3DOP_splits

### ISSUE THREAD

**Issue by PuJanhan** [WeijingShi/Point-GNN#53]
Thanks for your great work, I have a question about the 3DOP_splits . When I download the link(https://xiaozhichen.github.io/files/mv3d/imagesets.tar.gz) ,It contains test.txt, trian.txt, trainval.txt and val.txt  The names of the four files is not same with the description in 3DOP_splits .I dont konw the relationship between them. Could you tell me how to rename the file that I download?

**Comment by WeijingShi** [comment]
Hi @PuJanhan, The tar file should contain train.txt (the training split of kitti training set), val.txt (the val split of kitti training set), trainval.txt(the same as the kitting training set) and test.txt (the kitti test set). Beside those splits, we provide *_car.txt and *_pedestrian_cyclist.txt in the repository. For example, train_car.txt is a subset of train.txt, which contains cars. Thanks.
